### PR TITLE
release service-center 2.1.0

### PR DESCRIPTION
**Pull request by robotLJW**
【背景】发布 sc2.1.0 修改相关文档，修改相关文档

### CODE REVIEW

File: _release/cn/service_center_downloads.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ last_modified_at: 2021-06-04T00:50:43-55:00
...
...org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-windows-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-windows-amd64.tar.gz.sha512)| [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-linux-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-linux-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-linux-amd64.tar.gz.sha512)|[[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-darwin-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-darwin-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-darwin-amd64.tar.gz.sha512) | Java-Chassis 1.0.0 or above |
```
Comment (WillemJiang) [inline _release/cn/service_center_downloads.md:16]: we need use this url for the old file https://archive.apache.org/dist

File: _release/cn/service_center_downloads.md [hunk 2]
```diff
@@ -11,7 +11,8 @@ last_modified_at: 2021-06-04T00:50:43-55:00
...
...org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-windows-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-windows-amd64.tar.gz.sha512)| [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-linux-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-linux-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-linux-amd64.tar.gz.sha512)|[[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-darwin-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-darwin-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-darwin-amd64.tar.gz.sha512) | Java-Chassis 1.0.0 or above |
```
Comment (WillemJiang) [inline _release/cn/service_center_downloads.md:15]: we need use this url for the old file https://archive.apache.org/dist

File: _release/service_center_downloads.md [hunk 3]
```diff
@@ -4,15 +4,16 @@ lang: en
...
...org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-windows-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-windows-amd64.tar.gz.sha512)| [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-linux-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-linux-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-linux-amd64.tar.gz.sha512)|[[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-darwin-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-darwin-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.0.0/apache-servicecomb-service-center-2.0.0-darwin-amd64.tar.gz.sha512) | Java-Chassis 1.0.0 or above |
```
Comment (WillemJiang) [inline _release/service_center_downloads.md:16]: we need use this url for the old file https://archive.apache.org/dist

File: _release/service_center_downloads.md [hunk 4]
```diff
@@ -4,15 +4,16 @@ lang: en
...
...org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-windows-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-windows-amd64.tar.gz.sha512)| [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-linux-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-linux-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-linux-amd64.tar.gz.sha512)|[[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-darwin-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-darwin-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/1.3.0/apache-servicecomb-service-center-1.3.0-darwin-amd64.tar.gz.sha512) | Java-Chassis 1.0.0 or above |
```
Comment (WillemJiang) [inline _release/service_center_downloads.md:17]: we need use this url for the old file https://archive.apache.org/dist

**Review by WillemJiang — CHANGES_REQUESTED** [review]

**Review by little-cui — APPROVED** [review]

File: _release/cn/service_center_downloads.md [hunk 5]
```diff
@@ -11,7 +11,8 @@ last_modified_at: 2021-06-04T00:50:43-55:00
...
.../dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-windows-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-windows-amd64.tar.gz.sha512) | [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-linux-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-linux-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-linux-amd64.tar.gz.sha512) | [[Binary]](https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-darwin-amd64.tar.gz) [[asc]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-darwin-amd64.tar.gz.asc) [[sha512]](https://www.apache.org/dist/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-darwin-amd64.tar.gz.sha512) | Java-Chassis 1.0.0 or above |
```
Comment (WillemJiang) [inline _release/cn/service_center_downloads.md:14]: I just tried to access the url and got this link. https://dlcdn.apache.org/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-src.tar.gz

Comment (robotLJW) [reply _release/cn/service_center_downloads.md:14]: should be https://apache.org/dyn/closer.cgi/servicecomb/servicecomb-service-center/2.1.0/apache-servicecomb-service-center-2.1.0-src.tar.gz

**Review by WillemJiang — APPROVED** [review]

**Pull request merged**